hal/halx86/generic/sysinfo.c

```diff
@@ -44,7 +104,6 @@ HaliQuerySystemInformation(IN HAL_QUERY_INFORMATION_CLASS InformationClass,
 		REPORT_THIS_CASE(HalCmcLogInformation);
 		REPORT_THIS_CASE(HalCpeLogInformation);
 		REPORT_THIS_CASE(HalQueryMcaInterface);
-		REPORT_THIS_CASE(HalQueryAMLIIllegalIOPortAddresses);
```

Comment:
I'd suggest to move the implementation to this line, instead of just removing it.

Reply:
Fixed.